cloudbuild.yaml

```diff
@@ -37,6 +37,16 @@ steps:
     - 'REPO_NAME=${REPO_NAME}'
     - 'COMMIT_SHA=${COMMIT_SHA}'
 
+# Run authorization logic tests 
+- id: 'auth_logic_tests'
+  waitFor: ['build-image']
+  name: '${_DOCKER_IMAGE}'
+  entrypoint: 'bash'
+  args: [
+    '-l', '-c', 'cargo', 'test', '--manifest-path=/rust/tools/authorization-logic/'
```

Comment:
Add a comment as to why `-l` and `-c` are needed?

Reply:
It's a script that doesn't use these options now.